src/lib/components/QRCode.svelte

```diff
@@ -0,0 +1,115 @@
+<script lang="ts">
+  /**
+   * Inspired by:
+   *
+   * - Shoelace: https://github.com/shoelace-style/shoelace/blob/next/src/components/qr-code/qr-code.ts
+   * - DeckDeckGo: https://github.com/deckgo/deckdeckgo/blob/main/webcomponents/elements/src/components/qrcode/qrcode/qrcode.tsx
+   */
+  import { afterUpdate, createEventDispatcher } from "svelte";
+  import QrCreator from "qr-creator";
+  import { debounce } from "$lib/utils/debounce.utils";
+
+  export let ariaLabel: string | undefined = undefined;
+  export let value: string;
+
+  // Valid CSS colors
+  export let fillColor = "black";
+  export let backgroundColor = "white";
+
+  export let radius = 0;
```

Comment:
What does `radius` do?

Reply:
Added a comment.

> // The edge radius of each module. Must be between 0 and 0.5.

Did not implement particular rule. We can do that later if we need but currently we do not use it.